### PR TITLE
[MLDB-1104] merged the bucketize query fields into one

### PR DESCRIPTION
**Pull request by guyd**

### CODE REVIEW

File: plugins/bucketize_procedure.cc [hunk 1]
```diff
@@ -127,12 +117,10 @@ run(const ProcedureRunConfig & run,
 {
     SqlExpressionMldbContext context(server);
 
-    auto boundDataset = procedureConfig.inputDataset->bind(context);
+    auto boundDataset = procedureConfig.inputData.stm->from->bind(context);
 
     SelectExpression select(SelectExpression::parse("1"));
     vector<shared_ptr<SqlExpression> > calc;
-    ssize_t offset = 0;
```
Comment (guyd) [inline plugins/bucketize_procedure.cc]: @FinchPowers Not sure why passing an offset and limit was not allowed in bucketize.  Now this is allowed so if there  is a good reason for not allowing it, let me know.

**Comment by steadtler** [comment]
+ 1

**Comment by guyd** [comment]
C'est un + 1 ou un - 1? :-)

**Comment by steadtler** [comment]
whaaaat? c'etait un +1!

**Pull request merged**